### PR TITLE
Detailed console reporter

### PR DESCRIPTION
**Pull request by dpfg**

### CODE REVIEW

File: reporter.go [hunk 1]
```diff
@@ -34,60 +44,181 @@ func (r *ConsoleReporter) Init() {
...
+func (r *ConsoleReporter) verbose() bool {
+	return r.LogHTTP
+}
+
+func (r *ConsoleReporter) StartLine() {
+	r.Writer.Write([]byte("\n"))
+	r.Writer.Write([]byte(strings.Repeat(" ", r.IntendSize)))
+}
+
+func (r *ConsoleReporter) Intend() {
```
Comment (kajf) [inline reporter.go]: perhaps, should be inDenT not inTenD

Comment (dpfg) [reply reporter.go]: Thanks, fixed.

Comment (dpfg) [reply reporter.go]: Fixed

File: reporter.go [hunk 2]
```diff
@@ -34,60 +44,181 @@ func (r *ConsoleReporter) Init() {
...
+		}
+
+		r.Write(" ").Write(result.Case.Name)
+		r.Write(" [").Write(result.ExecFrame.Duration().Round(time.Millisecond)).Write("]")
+
+		if result.hasError() || r.LogHTTP {
+			for _, trace := range result.Traces {
+				r.Intend()
+
+				r.StartLine()
```
Comment (kajf) [inline reporter.go]: maybe enable chaining on StartLine()
for r.StartLine().Write()

**Pull request merged**